app/panel/components/Stats.jsx

```diff
@@ -517,7 +517,7 @@ class Stats extends React.Component {
 		return selectionData;
 	}
 
-	_isPlus = props => props.user && props.user.subscriptionsPlus;
+	_isPlus = props => props.user && props.user.plusAccess;
```

Comment:
Similarly, let's rename this helper to `_hasPlusAccess`

Reply:
Renamed!